ChangeLog.d/restore-x509-functions-to-public.txt

```diff
@@ -0,0 +1,6 @@
+Bugfix
```

Comment:
Thanks for not forgetting the ChangeLog, but this time I don't think an entry is needed, considering the bug was never present in any released version. (And the ChangeLog is about changes between say 3.5.2 and 3.6.0, not transient states in `development` in the meantime.)